ios/FluentUI/Vnext/List/ListHeaderFooter.swift

```diff
@@ -6,26 +6,27 @@
 import UIKit
 import SwiftUI
 
-/// View for List Header (availble for iOS 14.0+)
 struct Header: View {
-    let title: String
-    var tokens: MSFListTokens
+    let state: MSFListSectionState
+    var tokens: MSFListHeaderFooterTokens
 
-    init(title: String, tokens: MSFListTokens) {
-        self.title = title
-        self.tokens = tokens
+    init(state: MSFListSectionState) {
+        self.state = state
+        self.tokens = MSFListHeaderFooterTokens(style: state.style)
     }
 
     var body: some View {
```

Comment:
You need the modifier to retrieve the current theme in case there is an override for the window or container control:
This applies for every SwiftUI View that has a tokens property. (meanwhile I'm trying to find a way to avoid this duplication of code and yet affect all of the controls we need)

```
.onAppear {
                // When environment values are available through the view hierarchy:
                //  - If we get a non-default theme through the environment values,
                //    we use to override the theme from this view and its hierarchy.
                //  - Otherwise we just refresh the tokens to reflect the theme
                //    associated with the window that this View belongs to.
                if theme == ThemeKey.defaultValue {
                    self.tokens.updateForCurrentTheme()
                } else {
                    self.tokens.theme = theme
                }
            }
```